static/app/views/replays/replays.tsx

```diff
@@ -251,6 +251,9 @@ const ReplayUserBadge = styled(UserBadge)`
   font-size: ${p => p.theme.fontSizeLarge};
   font-weight: 400;
   line-height: 1.2;
+  div[class*='StyledEmail'] {
```

Comment:
This won't work in production as the component names are only there in development. You'll need to export `StyledEmail` (make sure leave a comment why you're exporting it) and import here where you can use it as a selector:

```suggestion
  ${StyledEmail} {
```

Reply:
Ah didn't know that, I'll get that adjusted.

Reply:
Actually if I'm waiting for Jesse to give feedback, is it worth doing this yet? I feel like it's prudent to hold off until I get feedback.

Reply:
I went ahead and updated this for now, if I have to roll it back no big deal I'd rather have the all the feedback addressed now.